### ISSUE TITLE
Grant commit permission

### ISSUE THREAD

**Issue by Bhagyashri-Sapnar** [jenkins-infra/helpdesk#3520]
### GitHub repositories

https://github.com/jenkinsci/qualys-cs-plugin
https://github.com/jenkinsci/qualys-pc-plugin
https://github.com/jenkinsci/qualys-was-plugin

### GitHub users to have permission

@Bhagyashri-Sapnar

### Permission level

Maintain

### Justification

New engineer on boarded on team

**Comment by NotMyFault** [comment]
cc @pmgupte @nikita-bedmutha @qualys-plugins

**Comment by lemeurherve** [comment]
@pmgupte @nikita-bedmutha @qualys-plugins gentle ping 🙂

**Comment by pmgupte** [comment]
I tried adding @Bhagyashri-Sapnar to the team, but it shows message saying she is not part of the organization. Please refer to the attached screenshot. 
<img width="866" alt="image" src="https://user-images.githubusercontent.com/3715755/235126549-c113d41b-07aa-4a77-a3a0-9112bbfd016b.png">

Any idea why so @halkeye @lemeurherve ?

**Comment by NotMyFault** [comment]
You can't invite users to the organization. I sent the user an invitation to the tree repository teams.

**Comment by pmgupte** [comment]
> You can't invite users to the organization. I sent the user an invitation to the tree repository teams.

Ah! That's the reason! @Bhagyashri-Sapnar I can add you the the team that maintains these plugins only after you are added to the parent org. Let me know once you become member of Jenkinsci org. 

@NotMyFault From this ticket, I assumed @Bhagyashri-Sapnar is already made member of the org, and next step is to add her to the team. Things are clear now, and I will await Bhagyashri to accept the invite.

**Comment by NotMyFault** [comment]
> and next step is to add her to the team

That is already done. You can send invites to teams too.
